rkt/gc.go

```diff
@@ -194,44 +237,14 @@ func deletePod(p *pod) {
 			return
 		}
 		defer s.Close()
-		stage1TreeStoreID, err := p.getStage1TreeStoreID()
-		if err != nil {
-			stderr("Error getting stage1 treeStoreID: %v", err)
-			return
-		}
-		stage1RootFS := s.GetTreeStoreRootFS(stage1TreeStoreID)
 
-		// execute stage1's GC
-		if err := stage0.GC(p.path(), p.uuid, stage1RootFS, globalFlags.Debug); err != nil {
-			stderr("Stage1 GC of pod %q failed: %v", p.uuid, err)
-			return
-		}
-
-		if p.usesOverlay() {
-			apps, err := p.getApps()
-			if err != nil {
-				stderr("Error retrieving app hashes from pod manifest: %v", err)
-				return
+		if err := cleanExitedGarbage(p, s); err != nil {
+			if globalFlags.Debug {
```

Comment:
I think of debug as something you can use to get more information about something that failed. For example, if `rkt run` fails, you run it with debug to see what's going on.

In this case, unless the user runs gc with `--debug` in the first place for some reason, she won't notice anything and the pod will be forcibly removed...